application/util/uri-helper.js

```diff
@@ -28,9 +30,25 @@ module.exports = {
      * @return {String}        URI with replaced parameter
      */
     injectValueIntoUri : function(name, uri, value) {
-        var regex = new RegExp(':' + name + '(?![A-z])');
+        console.log(value);
+        // inject array differently using name[]=value
+        if (typeof value ==='object') {
+            // skip if someone tries to put an object in a uri
```

Comment:
Perhaps we should `throw` if it's an object that isn't an array instead of failing silently. (That would essentially be a `LogicException` -- so we'd want to have a loud indication that something's wrong.)